hcs-sxc-java-plugins/hcs-sxc-java-plugins-cryptography/src/main/java/com/hedera/hcs/sxc/plugin/cryptography/keyrotation/KeyRotation.java

```diff
@@ -1,4 +1,4 @@
-package com.hedera.hcs.sxc.cryptography;
+package com.hedera.hcs.sxc.plugin.cryptography.keyrotation;
```

Comment:
Plugin name should indicate which key rotation scheme is being used to allow for others in the future.

Reply:
TODO, creating issue